apps/staking/src/components/OracleIntegrityStaking/index.tsx

```diff
@@ -517,14 +518,7 @@ const Publisher = ({
           </Meter>
         </PublisherTableCell>
         <PublisherTableCell className="text-center">
-          <div>
-            {calculateApy(
-              publisher.poolCapacity,
-              publisher.poolUtilization,
-              publisher.isSelf,
-            )}
-            %
-          </div>
+          <div>{publisher.lastApy}%</div>
```

Comment:
Nice but you should also change https://github.com/pyth-network/pyth-crosschain/blob/main/apps/staking/src/components/OracleIntegrityStaking/index.tsx#L177-L180 to use this new field

Note as I mentioned above we still need a method to calculate the apy for https://github.com/pyth-network/pyth-crosschain/blob/main/apps/staking/src/components/OracleIntegrityStaking/index.tsx#L646-L686

Reply:
both use calculateApy now